buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/UpdateIntellijVersions.kt

```diff
@@ -0,0 +1,104 @@
+package com.virtuslab.gitmachete.buildsrc
+
+import com.virtuslab.gitmachete.buildsrc.IntellijVersionHelper.getMajorPart
+import com.virtuslab.gitmachete.buildsrc.IntellijVersionHelper.toBuildNumber
+import com.virtuslab.gitmachete.buildsrc.IntellijVersionHelper.versionIsNewerThan
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.Internal
+import org.gradle.api.tasks.TaskAction
+import org.jsoup.Jsoup
+import org.jsoup.select.Elements
+
+open class UpdateIntellijVersions : DefaultTask() {
+
+  @get:Internal
+  val intellijReleasesContents: Elements by lazy {
+    getLinksFromUrl("https://www.jetbrains.com/intellij-repository/releases/")
+  }
+
+  @get:Internal
+  val intellijSnapshotsContents: Elements by lazy {
```

Comment:
Just `private val`? (now that `buildSrc/` is excluded from `scripts/prohibit-kotlin-private-fields`)